sound/soc/sof/pcm.c

```diff
@@ -426,8 +426,12 @@ static int sof_pcm_open(struct snd_pcm_substream *substream)
 			  SNDRV_PCM_INFO_MMAP_VALID |
 			  SNDRV_PCM_INFO_INTERLEAVED |
 			  SNDRV_PCM_INFO_PAUSE |
-			  SNDRV_PCM_INFO_RESUME |
-			  SNDRV_PCM_INFO_NO_PERIOD_WAKEUP;
+			  SNDRV_PCM_INFO_RESUME;
+
+#if !IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_FORCE_ENABLE_WAKEUP)
```

Comment:
what is _FORCE_ENABLE_WAKEUP mean? here we only claim that the driver support no_irq mode, no need any config item value check.